csharp-scenarios/01-Alien Translator.dib

```diff
@@ -0,0 +1,214 @@
+
+Instead of replacing every *myName*, let’s create a string variable called name that we can assign to a different name one time and it will update all the myNames throughout the introduction! Now, replace all the instances of *myName* with our new variable, *name*. 
+
+#!csharp
+
+string name = "myName";
+
+Console.WriteLine("Hi extraterrestrials, my name is myName."); 
+Console.WriteLine("myName is a researcher."); 
+Console.WriteLine("myName studies language."); 
+
+#!markdown
+
+Now, you can set name to the name of whoever is being introduced. Try replacing *myName* with the name Joon and running it.  
+
+The above lines with Console.WriteLine are **string literals**, which means it is text that represents a constant string. 
+
+#!markdown
+
+## Let's translate a message
+It’s time to start translating. Your linguistic partner gives you the following graph that includes all the words that she’s been able to translate in language X and their English translation. We’re going to use this data to translate the alien messages. 
+
+### Language data
+This data is organized into a [switch expression](https://docs.microsoft.com/dotnet/csharp/language-reference/operators/switch-expression). This **switch statement** takes in an alien word as input, and outputs the English translation. In general, a switch expression matches the input against a series of cases and the output is dependent upon which case is matched. In order to translate the message, we’ll use this switch expression to help us match the alien words and receive the proper English word back. 
+
+#!csharp
+
+String ToEnglish(string word) => word switch
+{
+    "yoish"     => "the aliens ",
+    "shwaye"    => "crashed ",
+    "ni"        => "on ",
+    "ceroyash"  => "Earth ",
+    "oppor"     => "need ",
+    "shioy"     => "humans ",
+    "abree"     => "help ",
+    "coierm"    => "to go ",
+    "veiesht"   => "home ",
+    "shir"      => "can ",
+    "argsha"    => "spaceship ",
+    "lleog"     => "dirt ",
+    "afala"     => "trees ",
+    "eshteve"   => "walk ",
+    "blarg"     => "speak ",
+    "efellesh"  => "peace ",
+    "nomshce"   => "want ",
+    "warshel"   => "please ",
+    "yoishel"   => "Alien planet ",
+    _           => "Unknown "
+};
+
+#!markdown
+
+Looks like the aliens are sending their first message:
+### "warshel abree yoish" 
+
+#!markdown
+
+For our first translation, let’s replace each language X word with its English word counterpart. To do this, we’ll use the [String.Replace method](https://docs.microsoft.com/dotnet/api/system.string.replace?view=net-6.0). To use this method correctly, we pass in the string we want to replace and the replacement string. Run the cell below to see how it works. 
+
+Here's an example:
+
+#!csharp
+
+String stringName = “He is cold”; 
+
+// Below, we replace "He" with "She" and "cold" with "cool"
+stringName.Replace("He”, "She”).Replace(“cold”,”cool”); 
```

Comment:
so here it should be:
```
stringName = stringName.Rep....
```

it doesnt' to it inline. THen you should print it out as well after so they see it.